### DISCUSSION TITLE
Can't boot form img.

### DISCUSSION THREAD

**Discussion by caiiiycuk** [copy/v86#1171] [Q&A]
Hi I have a hdd with Windows 95 in raw format. It works just fine in dosbox-x, I can boot from it and work. But when I use it with v86 it says "Invalid System Disk". 

![изображение](https://github.com/user-attachments/assets/f6e8a912-55aa-4814-97d5-26e59a8d956b)

dosbox-x mount command:
```
imgmount 2 c.img -size 512,63,4,975 -t hdd -fs none
```

qemu-img info:
```
image: c.img
file format: raw
virtual size: 120 MiB (125798400 bytes)
disk size: 120 MiB
Child node '/file':
    filename: c.img
    protocol type: file
    file length: 120 MiB (125798400 bytes)
    disk size: 120 MiB
```

file info:
```
c.img: DOS/MBR boot sector MS-MBR,D0S version 3.3-7.0 english at offset 0x8b "Invalid partition table" at offset 0xa3 "Error loading operating system" at offset 0xc2 "Missing operating system", disk signature 0xe3657373, created with driveID 0x80 at 11:19:50, FREE-DOS MBR; partition 1 : ID=0x6, active, start-CHS (0x0,1,1), end-CHS (0x3ce,3,63), startsector 63, 245637 sectors
```

Can I use it somehow?

**Comment by ProgrammerIn-wonderland** [comment]
What happens if you do qemu-system-i386 -hda yourimage.img

**Reply by caiiiycuk** [reply]
It start booting but failing in the middle.

![изображение](https://github.com/user-attachments/assets/0445907a-10d2-4972-aa1f-323ba27602c4)

**Comment by YD711YT** [comment]
You have too much RAM( can’t be over 480MB)
